sources/platform/actors/publishing/testing.mdx

```diff
@@ -9,17 +9,27 @@ sidebar_position: 3
 
 ---
 
+### Why we test
+We want to make sure that all Actors in the Store are top notch, or at least as top notch they can be. Since there are many of them, we have an automated testing procedure in place that tests all Actors daily.
```

Comment:
```suggestion
We want to make sure that all Actors in Apify Store are top-notch, or at least as top-notch as they can be. Since there are many of them, we have an automated testing procedure in place that tests all Actors daily.
```